src/module_1/module_1_meteo_api.py

```diff
@@ -39,25 +50,28 @@ def call_api(url):
             logging.error(f"\nAPI request failed with Code: {response.status_code}")
             return None
     except requests.exceptions.RequestException as exception:
-            logging.error(f"\nAPI request failed with Exception: {exception}")
-            return None
+        logging.error(f"\nAPI request failed with Exception: {exception}")
+        return None
 
 
 def get_data_meteo_api(city, from_year, until_year):
     coordinates = COORDINATES.get(city)
```

Comment:
Nitpick: I prefere to do `if city not in COORDINATES`

Reply:
Solved. Please go ahead with future nitpicks 🤓